demo/apr_io_demo.py

```diff
@@ -3,62 +3,37 @@
 from skimage import io as skio
 
 
-def main():
-    """
-    This demo converts a selected TIFF image to an APR, writes the result to file and then reads the file.
-    """
+"""
+This demo converts a selected TIFF image to an APR, writes the result to file and then reads the file.
+"""
 
-    io_int = pyapr.filegui.InteractiveIO()
+io_int = pyapr.filegui.InteractiveIO()
 
-    # Read in an image
-    fpath = io_int.get_tiff_file_name()
-    img = skio.imread(fpath).astype(np.uint16)
+# Read in an image
+fpath = io_int.get_tiff_file_name()
+img = skio.imread(fpath).astype(np.uint16)
```

Comment:
Do we now need this casting to uint16? I thought the below code now allowed for it to be arbitraty types?

Reply:
ah, no we don't. it can be uint8, uint16 or f32, other types will result in errors.

actually I think we can remove this demo? I guess the point was to show the read/write API, but this is there in the other scripts

Reply:
removed the cast in 1c87b50afddc80b1e78a566ab165175fbd4f898c
guess it doesn't hurt to keep the demo for now